application/src/test/java/org/thingsboard/server/service/sync/ie/ExportImportServiceSqlTest.java

```diff
@@ -145,6 +156,8 @@ public class ExportImportServiceSqlTest extends AbstractControllerTest {
     protected TenantService tenantService;
     @Autowired
     protected EntityViewService entityViewService;
+    @Autowired
+    protected CalculatedFieldService calculatedFieldService;
 
```

Comment:
Add tests to VersionControlTest - import within single tenant, and between two tenants